docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  db:
+    image: mdillon/postgis:9.4
+    ports:
+      - "5432"
+  ashlar-server:
+    build: ./ashlar-server
+    links:
+      - db
+    depends_on:
+      - db
+    volumes:
+      - ./ashlar-server:/opt/ashlar-server
+    command: /bin/bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```

Comment:
Putting the migrations here is a good instinct but it could get you into trouble down the road if you need to manually manage migrations and docker-compose keeps migrating forward every time you restart. We use a convention called Scripts-To-Rule-Them-All that I think will be helpful here. Our documentation for that is in a private repo so I'll send it to you via Slack.

Reply:
Didn't think of that! I'll move this logic to `scripts/update`.